_posts/2021-03-05-scala-3-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+  all the changes in Scala 3, and the rest were concerned with bugs 
+  or outlined the internal thought process of the respondent. 
+
+### Tooling
+We can identify several categories that are important for Scala 3:
+- Build tools: sbt remains the most used build tool and the most critical for Scala 3 adoption. 
+  Nevertheless, all build tools are important and need to support Scala 3. The comments also 
+  emphasize this with 5 comments reporting the criticality of Maven, Gradle or Bazel support.  
+- IDE support: IntelliJ comes second in the survey. Comments emphasize the need of a great 
+  IDE experience for the adoption of Scala 3, and specifically IntelliJ one.
+- Formatting and linting tools: Scalafmt and Scalafix are the most used and most 
+  reported as very important in this survey. The comments bring attention 
+  also to other tools like Scapegoat or Scalariform.
+Other: Scala.js, Scala Native and Ammonite.
+
+### Migration
+The survey shows that the individual members of the community are 
+more concerned about the availability of their library dependencies 
+than they are concerned about the migration of their own code base. 
+Indeed it will require some time and effort from the community to progressively move the existing ecosystem to Scala 3.
+
+While in most cases the migration will not be harder than it 
+was between 2.11, 2.12 and 2.13, special care should be taken for 
+translating the macro libraries and for replacing the available macro annotations. 
+The success of this undertaking will depend on the quality of the available documentation and resources for learning.
+
+The survey points out that better-monadic-for is an important piece 
+of software that is currently missing in Scala 3. Some compiler options, 
+such as `-Xlint`, `-Yimports`, `-V` and `-W` options are not crucial but they are nonetheless very useful.
+
+Last but not least, the survey reminds us of the importance of compile 
+time and runtime performance. They are essential to the success of Scala 3.
+
+### Communication
+Things that stood out clearly:
+- scala_lang.org is the most used source of information
+- Even though 75% of responders responded that they don’t use newsletters/emails as their source, 60 people
+  commented that they do appreciate one specific newsletter: ScalaTimes
+- Twitter showed up to be the most used source (also judging by comments) 
+  to stay updated on novelties in Scala.
+- The survey respondents expressed wide opinions about their platform preferences 
+  as well as their “habitats”, ranging from, for example: “... No time for more newsletters, 
+  my email inbox is flooded already” vs “... I think it's important to notice that 
+  there are still people out there not using social media to stay up to date about tech stuff.”
+
+Based on the overall survey, we identified topics that should be more prominent and better followed up with, see other sections.
```

Comment:
not sure what the intended meaning of "followed up with" is, here. do you mean "followed up on"?